content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -212,46 +212,48 @@ the current value.
 
 Finally, right before HPA scales the target, the scale recommendation is recorded. The
 controller considers all recommendations within a configurable window choosing the
-highest recommendation from within that window. This value can be configured using the
-`--horizontal-pod-autoscaler-downscale-stabilization` flag, which defaults to 5 minutes.
+highest recommendation from within that window. You can configure this value using the
+`--horizontal-pod-autoscaler-downscale-stabilization` command line option, which defaults to 5 minutes.
 This means that scaledowns will occur gradually, smoothing out the impact of rapidly
 fluctuating metric values.
 
-### Pod readiness and autoscaling metrics
+## Pod readiness and autoscaling metrics
 
-The HorizontalPodAutoscaler (HPA) controller includes two flags that influence how CPU metrics are collected from Pods during startup:
+The HorizontalPodAutoscaler (HPA) controller includes two command line options that influence how CPU metrics are collected from Pods during startup:
 
 1. `--horizontal-pod-autoscaler-cpu-initialization-period` (default: 5 minutes)
 
   This defines the time window after a Pod starts during which its **CPU usage is ignored** unless:
     - The Pod is in a `Ready` state **and**
     - The metric sample was taken entirely during the period it was `Ready`.
 
-  This flag helps **exclude misleading high CPU usage** from initializing Pods (e.g., Java apps warming up) in HPA scaling decisions.
+  This command line option helps **exclude misleading high CPU usage** from initializing Pods (for example: Java apps warming up) in HPA scaling decisions.
 
-2. `--horizontal-pod-autoscaler-initial-readiness-delay` (default: 30 seconds)
+1. `--horizontal-pod-autoscaler-initial-readiness-delay` (default: 30 seconds)
 
   This defines a short delay period after a Pod starts during which the HPA controller treats Pods that are currently `Unready` as still initializing, **even if they have previously transitioned to `Ready` briefly**.
 
   It is designed to:
     - Avoid including Pods that rapidly fluctuate between `Ready` and `Unready` during startup.
     - Ensure stability in the initial readiness signal before HPA considers their metrics valid.
 
-**Key behaviors:**
+
+You can only set these command line options cluster-wide.
+
+### Key behaviors for pod readiness {#pod-readiness-key-behaviors}
+
 - If a Pod is `Ready` and remains `Ready`, it can be counted as contributing metrics even within the delay.
 - If a Pod rapidly toggles between `Ready` and `Unready`, metrics are ignored until it’s considered stably `Ready`.
 
-#### Best Practice:
+### Good practise for pod readiness {#pod-readiness-good-practises}
```

Comment:
I don't believe this change is correct -- if we're defaulting to British English rather than American English, "practice" is the noun and "practise" is the verb. In this case, it's a noun. American English uses "practice" for both.